docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_keyvault_extension/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+type: docs
+title: "Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions"
+linkTitle: "Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions"
+weight: 3
+description: >
+---
+
+## Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions
+
+The following README will guide you on how to enable [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/overview) for a Cluster API that is projected as an Azure Arc connected cluster.
+
+In this guide, you will hook the Cluster API to Azure Key Vault by deploying the [Azure Key Vault extension](https://docs.microsoft.com/en-us/azure/aks/csi-secrets-store-driver) and a sample app on your Kubernetes cluster in order to integrate Azure Key Vault as a secrets store with a Kubernetes cluster via a CSI volume. This will also include deploying a Key Vault and sample secret on your Azure subscription.
```

Comment:
```suggestion
In this guide, you will hook the Cluster API to Azure Key Vault by deploying the [Azure Key Vault extension](https://docs.microsoft.com/en-us/azure/aks/csi-secrets-store-driver) and a sample app on your Kubernetes cluster in order to integrate Azure Key Vault as a secrets store with a Kubernetes cluster via a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) volume. This will also include deploying a Key Vault and sample secret on your Azure subscription.
```

Reply:
Should we update _Azure Key Vault extension_ link to either extension public documentation or upstream docs? This link is referring AKS add-on and might be confusing to user.